assignments/function-conversion.js

```diff
@@ -5,26 +5,52 @@
 // };
 // myFunction();
 
+let myFunction = () => {
+  console.log("Function was invoked!");
+};
+myFunction();
+
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {
+   return param;
```

Comment:
You wouldn't have to return here since it is implied due to the fact that you only have one method